x-pack/plugins/rule_registry/server/utils/create_persistence_rule_type_wrapper.ts

```diff
@@ -307,7 +301,7 @@ export const createPersistenceRuleTypeWrapper: CreatePersistenceRuleTypeWrapper
                   alerts: enrichedAlerts,
                   options,
                   kibanaVersion: ruleDataClient.kibanaVersion,
-                  currentTimeOverride,
+                  currentTimeOverride: undefined,
```

Comment:
Do we need to set this manually to undefined?

Reply:
We just revert how it was, for this case it probably don't change if we will not pass it, but maybe initial idea was to make it very explicit